### PR TITLE
changed readme

### PR DESCRIPTION
**Pull request by DerThorsten**
The dev install instuctions are incomplete. It misses `pip install jupyterite`.
I also added cnda command to create a fresh env

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,8 +38,12 @@ pip uninstall jupyterlite-p5-kernel
 # Clone the repo to your local environment
 # Change directory to the fork directory
 
-# Install JupyterLab
-python -m pip install jupyterlab
+# create a new enviroment
+conda create --name p5-kernel
```
Comment (jtpio) [inline README.md:42]: Or we can also advertise `mamba` :)

```suggestion
mamba create --name jupyterlite-p5-kernel -c conda-forge python=3.9 yarn jupyterlab
```

File: README.md [hunk 2]
```diff
@@ -54,11 +58,14 @@ jlpm run build
...
 ```bash
+# install jupyterlite
+python -m pip install jupyterlite
+
 # build a new JupyterLite site
 jupyter lite build
 
 # server the site
-jupyter lite serve
+jupyter lite serveatus
```
Comment (jtpio) [inline README.md:68]: Looks like a typo?

Comment (DerThorsten) [reply README.md]: jop....this typo sneaked in

**Review by jtpio — APPROVED** [review]
Thanks!

**Pull request merged**